File: fix-branding.py
import os
import glob

REPLACEMENTS = [
    ('百度', 'WinGo'),
    ('清华', 'WinGo'),
    ('OpenMAIC', 'WinGo'),
    ('THU-MAIC', 'WinGo'),
]

base_path = '/app/.next/server'
files = glob.glob(base_path + '/**/*.js', recursive=True)

for f in files:
    try:
        with open(f, 'r', encoding='utf-8', errors='ignore') as fh:
            content = fh.read()
        new_content = content
        for old, new in REPLACEMENTS:
            new_content = new_content.replace(old, new)
        if new_content != content:
            with open(f, 'w', encoding='utf-8') as fh:
                fh.write(new_content)
            print(f'Fixed: {f}')
    except Exception as e:
        print(f'Error {f}: {e}')

print('Done.')

File: fix-branding.js
const fs = require('fs');
const path = require('path');

const REPLACEMENTS = [
  ['百度', 'WinGo'],
  ['清华', 'WinGo'],
  ['OpenMAIC', 'WinGo'],
  ['THU-MAIC', 'WinGo'],
];

function fixDir(dir) {
  const entries = fs.readdirSync(dir, { withFileTypes: true });
  for (const entry of entries) {
    const fullPath = path.join(dir, entry.name);
    if (entry.isDirectory()) {
      fixDir(fullPath);
    } else if (entry.name.endsWith('.js')) {
      let content = fs.readFileSync(fullPath, 'utf-8');
      let changed = false;
      for (const [old, newStr] of REPLACEMENTS) {
        if (content.includes(old)) {
          content = content.split(old).join(newStr);
          changed = true;
        }
      }
      if (changed) {
        fs.writeFileSync(fullPath, content, 'utf-8');
        console.log('Fixed:', fullPath);
      }
    }
  }
}

fixDir('/app/.next/server');
console.log('Done.');
